lib/github_backend.rb

```diff
@@ -232,10 +232,10 @@ def get_repos(opts)
 		if opts.repos != nil
 			repos = repos.concat(opts.repos)
 		end
-		if opts.orgas != nil
-			opts.orgas.each do |orga|
+		if !opts.orgs.nil?
+		  opts.orgs.each do |org|
 				begin
-					repos = repos.concat(request('org_repos', [orga, {:type => 'owner'}]).map {|repo|repo.full_name.dup})
+					repos = repos.concat(request('org_repos', [org, {:type => 'owner'}]).map {|repo|repo.full_name.dup})
```

Comment:
Line is too long. [105/100]<br>Use the new Ruby 1.9 hash syntax.<br>Use 2 (not 1) spaces for indentation.<br>Space after closing `|` missing.<br>Space between { and | missing.<br>Space missing inside }.<br>Space inside { missing.<br>Space inside } missing.<br>Tab detected.